features/creating_order_with_online_payment.feature

```diff
@@ -0,0 +1,26 @@
+@managing_orders
+Feature: Creating order with online payment
+    In order to have order placed by Admin and paid by Customer
```

Comment:
`In order to place order in he name of a Customer and forward the payment to them`? wdyt?